<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoardSettingsWidget</class>
 <widget class="QWidget" name="BoardSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>501</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="m_viewerLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="m_pieceSetLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Piece Set:</string>
          </property>
          <property name="buddy">
           <cstring>m_pieceSetCombo</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_pieceSetCombo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="m_pieceSizeLabel">
          <property name="toolTip">
           <string>Size of the pieces in relation to the squares</string>
          </property>
          <property name="text">
           <string>Pie&amp;ce Size:</string>
          </property>
          <property name="buddy">
           <cstring>m_pieceSizeSpin</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="m_pieceSizeSpin">
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="minimum">
           <number>65</number>
          </property>
          <property name="maximum">
           <number>99</number>
          </property>
          <property name="value">
           <number>80</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="m_lightSquareColorBtn">
        <property name="text">
         <string>Light Square Color:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_lightSquareColorEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxLength">
         <number>30</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="m_darkSquareColorBtn">
        <property name="text">
         <string>Dark Square Color:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_darkSquareColorEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxLength">
         <number>30</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="m_borderColorBtn">
        <property name="text">
         <string>Border Color:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_borderColorEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxLength">
         <number>30</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="m_coordTextColorBtn">
        <property name="text">
         <string>Coordinate Text Color:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_coordTextColorEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxLength">
         <number>30</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>m_pieceSetCombo</tabstop>
  <tabstop>m_pieceSizeSpin</tabstop>
  <tabstop>m_lightSquareColorBtn</tabstop>
  <tabstop>m_lightSquareColorEdit</tabstop>
  <tabstop>m_darkSquareColorBtn</tabstop>
  <tabstop>m_darkSquareColorEdit</tabstop>
  <tabstop>m_borderColorBtn</tabstop>
  <tabstop>m_borderColorEdit</tabstop>
  <tabstop>m_coordTextColorBtn</tabstop>
  <tabstop>m_coordTextColorEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
